docs/reference/reference-intro.mdx

```diff
@@ -46,4 +46,18 @@ slug: /reference
       </div>
     </div>
   </div>
+
+  <div class="col col--6">
+    <div class="card shadow">
+      <div class="card__body">
+        <h3>super.json</h3>
+        <p>
+          Primary configuration file for OneSDK.js.
+        </p>
+      </div>
+      <div class="card__footer">
+        <a target="_blank" href="/docs/reference/superjson" class="button button--primary button--block">super.json on GitHub</a>
```

Comment:
Preferably use the [Link component](https://docusaurus.io/docs/docusaurus-core#link) without target=_blank